wdio.conf.js

```diff
@@ -100,15 +100,15 @@ const config = {
   seleniumArgs: {
     drivers: {
       chrome: {
-        version: 2.31,
+        version: 'latest',
```

Comment:
@jeffswartz It seems `'latest'` is not a valid option since the API doesn't support requests for the latest version of the driver. I'm getting this error: 
`Error: Could not download https://chromedriver.storage.googleapis.com/latest/chromedriver_linux64.zip`

I suggest to change the value to the current latest version `2.46` (See: https://chromedriver.storage.googleapis.com/LATEST_RELEASE)